v22.1/export.md

```diff
@@ -57,11 +64,11 @@ You can control the [`EXPORT`](export.html) process's behavior using any of the
 
 Key                 | <div style="width:130px">Context</div> | Value                                                                                                                             |
 --------------------+-----------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------
-`delimiter`         |    `CSV DATA`   |  The ASCII character that delimits columns in your rows. If not using comma as your column delimiter, you can specify another ASCII character as the delimiter. **Default:** `,`. <br><br>To use tab-delimited values: `WITH delimiter = e'\t'`
-`nullas`            |    `CSV DATA`, `DELIMITED DATA`          |  The string that should be used to represent _NULL_ values. To avoid collisions, it is important to pick `nullas` values that do not appear in the exported data. <br><br>To use empty columns as _NULL_: `WITH nullas = ''`
-`compression`       |    `CSV DATA`   |  This instructs export to write `gzip` compressed CSV files to the specified destination. <br><br>See the [example](#export-gzip-compressed-csv-files) below.
-`chunk_rows`        |    `CSV DATA`   |  The number of rows to be converted to CSV and written to a single CSV file. **Default:** `100000`. <br>For example, `WITH chunk_rows = '5000'` for a table with 10,000 rows would produce two CSV files. <br><br>**Note**:`EXPORT` will stop and upload the CSV file whether the configured limit for `chunk_rows` or `chunk_size` is reached first.
-`chunk_size`        |    `CSV DATA`   | A target size per CSV file that you can specify during an `EXPORT`. Once the target size is reached, the CSV file is uploaded before processing further rows. **Default:** `32MB`. <br>For example, to set the size of each CSV file uploaded during the export to 10MB: `WITH chunk_size = '10MB'`. <br><br>**Note**:`EXPORT` will stop and upload the CSV file whether the configured limit for `chunk_rows` or `chunk_size` is reached first.
+`delimiter`         |    `CSV DATA`, `PARQUET DATA`   |  The ASCII character that delimits columns in your rows. If not using comma as your column delimiter, you can specify another ASCII character as the delimiter. **Default:** `,`. <br><br>To use tab-delimited values: `WITH delimiter = e'\t'`
+`nullas`            |    `CSV DATA`, `DELIMITED DATA`, `PARQUET DATA`   |  The string that should be used to represent `NULL` values. To avoid collisions, it is important to pick `nullas` values that do not appear in the exported data. <br><br>To use empty columns as `NULL`: `WITH nullas = ''`
+`compression`       |    `CSV DATA`, `PARQUET DATA`   |  This instructs export to write `gzip` compressed files to the specified destination. <br><br>See the [example](#export-gzip-compressed-files).
```

Comment:
nit: csv can do gzip compression, and parquet can do gzip _and_ `snappy` compression.